src/renderers/dom/client/__tests__/ReactMount-test.js

```diff
@@ -163,6 +163,27 @@ describe('ReactMount', function() {
     );
   });
 
+  it('should warn when component does not follow proper case convention',
+    function() {
+      var container = document.createElement('container');
+      var myDiv = <div>React</div>;
+      var mySvg = <svg><polygon points="-145.6 556.9 104.8-145.6 429"/></svg>;
+      var centered = React.createClass({
```

Comment:
I think the warning would fire even without this class being defined, right?  This is effectively just catching unknown elements?